.github/workflows/changelog.yml

```diff
@@ -28,6 +28,6 @@ jobs:
             echo "A CHANGELOG was modified. Looks good!"
           else
             echo "No CHANGELOG was modified."
-            echo "Please add a CHANGELOG entry, or add the \"Skip Changelog\" label if not required."
+            echo "Please add a CHANGELOG entry, or add \"[chore]\" to the title of the pull request if not required."
```

Comment:
```suggestion
            echo "Please add a CHANGELOG entry, or add either \"[chore]\" to the title of the pull request or add the \"Skip Changelog\" label if not required."
```

Reply:
I guess it depends if you want to specifically callout the PR is a chore for the sake of getting reviewers to quickly review it haha. Seeing [chore] in the title normally tells me I could review a PR relatively fast.